portal-impl/src/com/liferay/portlet/social/service/impl/SocialActivityCounterLocalServiceImpl.java

```diff
@@ -235,6 +235,13 @@ public void addActivityCounters(SocialActivity activity)
 				activity.getGroupId(), activity.getClassName(),
 				activity.getType());
 
+		if (activity.getType() == SocialActivityConstants.TYPE_REVOKE_VOTE) {
```

Comment:
Hey @hudakl @robertoDiaz 
I don't think we should have this logic. We need to treat REVOKE_VOTE same as ADD_VOTE and we shouldn't have any special condition or special logic for that. If we add the new activity in liferay-social.xml we shouldn't need to have this here, right? Or is there any other reason why this is needed?